src/views/sign-up-view/sign-up.module.css

```diff
@@ -120,27 +120,23 @@
 
 .asideGraphicContainer {
 	align-items: center;
-	justify-content: flex-end;
+	justify-content: center;
```

Comment:
This was done to make this graphic be centered with the text on "tablet sizes", here is how it looks now:

![Screenshot 2024-04-25 at 3 01 26 PM](https://github.com/hashicorp/dev-portal/assets/1957315/20a5a1e1-d9c6-49a8-9af8-6e23a977f442)